clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/BasicAmazonDeployHandler.groovy

```diff
@@ -256,7 +256,9 @@ class BasicAmazonDeployHandler implements DeployHandler<BasicAmazonDeployDescrip
         regionScopedProvider: regionScopedProvider,
         base64UserData: description.base64UserData,
         legacyUdf: description.legacyUdf,
-        tags: description.tags)
+        tags: description.tags,
+        maxSecurityGroupsPerInstance: account.getMaxSecurityGroupsPerInstance() ?: deployDefaults.maxSecurityGroups
```

Comment:
Caveat that if this rolls out we'll need to track down each account and make sure `maxSecurityGroupsPerInstance` is set right otherwise we might have grief with the default of 5.